src/pages/api/v1/user/interview-prep/sheet.ts

```diff
@@ -62,6 +67,14 @@ const handleMarkQuestionCompleted = async (
       );
     }
 
+    if (!isCompleted) {
+      await deductUserPointsFromDB(userId, 'COMPLETE_QUESTION');
+    }
+
+    if (isCompleted) {
+      await updateGamificationRecord(userId, 'COMPLETE_QUESTION');
+    }
+
```

Comment:
what's this code? Same thing wrotten twice. 

Why can't we keep things simple?